graphresponse.pb.go

```diff
@@ -0,0 +1,2552 @@
+import fmt "fmt"
+import math "math"
+
+import (
+	context "golang.org/x/net/context"
+	grpc "google.golang.org/grpc"
+)
+
+import io "io"
+
+// Reference imports to suppress errors if they are not otherwise used.
+var _ = proto.Marshal
+var _ = fmt.Errorf
+var _ = math.Inf
+
+// This is a compile-time assertion to ensure that this generated file
+// is compatible with the proto package it is being compiled against.
+// A compilation error at this line likely means your copy of the
+// proto package needs to be updated.
+const _ = proto.ProtoPackageIsVersion2 // please upgrade the proto package
+
+type NQuad struct {
+	Sub   string `protobuf:"bytes,1,opt,name=sub,proto3" json:"sub,omitempty"`
+	Pred  string `protobuf:"bytes,2,opt,name=pred,proto3" json:"pred,omitempty"`
+	ObjId string `protobuf:"bytes,3,opt,name=objId,proto3" json:"objId,omitempty"`
+	Value *Value `protobuf:"bytes,4,opt,name=value" json:"value,omitempty"`
+	Label string `protobuf:"bytes,5,opt,name=label,proto3" json:"label,omitempty"`
+}
+
+func (m *NQuad) Reset()                    { *m = NQuad{} }
+func (m *NQuad) String() string            { return proto.CompactTextString(m) }
+func (*NQuad) ProtoMessage()               {}
+func (*NQuad) Descriptor() ([]byte, []int) { return fileDescriptorGraphresponse, []int{0} }
+
+func (m *NQuad) GetValue() *Value {
+	if m != nil {
+		return m.Value
+	}
+	return nil
+}
+
+type Value struct {
+	// Types that are valid to be assigned to Val:
+	//	*Value_BytesVal
+	//	*Value_IntVal
+	//	*Value_BoolVal
+	//	*Value_StrVal
+	//	*Value_DoubleVal
+	//	*Value_GeoVal
+	Val isValue_Val `protobuf_oneof:"val"`
+}
+
+func (m *Value) Reset()                    { *m = Value{} }
+func (m *Value) String() string            { return proto.CompactTextString(m) }
+func (*Value) ProtoMessage()               {}
+func (*Value) Descriptor() ([]byte, []int) { return fileDescriptorGraphresponse, []int{1} }
+
+type isValue_Val interface {
+	isValue_Val()
+	MarshalTo([]byte) (int, error)
+	Size() int
+}
+
+type Value_BytesVal struct {
+	BytesVal []byte `protobuf:"bytes,1,opt,name=bytes_val,json=bytesVal,proto3,oneof"`
+}
+type Value_IntVal struct {
+	IntVal int32 `protobuf:"varint,2,opt,name=int_val,json=intVal,proto3,oneof"`
+}
+type Value_BoolVal struct {
+	BoolVal bool `protobuf:"varint,3,opt,name=bool_val,json=boolVal,proto3,oneof"`
+}
+type Value_StrVal struct {
+	StrVal string `protobuf:"bytes,4,opt,name=str_val,json=strVal,proto3,oneof"`
+}
+type Value_DoubleVal struct {
+	DoubleVal float64 `protobuf:"fixed64,5,opt,name=double_val,json=doubleVal,proto3,oneof"`
+}
+type Value_GeoVal struct {
+	GeoVal []byte `protobuf:"bytes,6,opt,name=geo_val,json=geoVal,proto3,oneof"`
+}
+
+func (*Value_BytesVal) isValue_Val()  {}
+func (*Value_IntVal) isValue_Val()    {}
+func (*Value_BoolVal) isValue_Val()   {}
+func (*Value_StrVal) isValue_Val()    {}
+func (*Value_DoubleVal) isValue_Val() {}
+func (*Value_GeoVal) isValue_Val()    {}
+
+func (m *Value) GetVal() isValue_Val {
```

Comment:
exported method GetVal returns unexported type graph.isValue_Val, which can be annoying to use